content/8.x/admin/Compatibility Matrix.adoc

```diff
@@ -16,14 +16,11 @@ The following documentation is for SuiteCRM Version 8+; to see documentation on
 
 2+^h| Platform 
 
-| Linux, Unix, Mac OS | Any version supporting PHP 
-
-| Windows | Windows Server 2012+
-
+| Linux, Unix, Mac OS | Any version supporting PHP
 | PHP | 7.3, 7.4, 8.0
 | Angular CLI | 11+
```

Comment:
Angular, Node, yarn are all dev dependencies, may put them in separate section

Reply:
all atomic commit if possible

Reply:
Whoops, though i had actually already done that, will update.
And will rebases to fix the commit issue

Reply:
Should be good for review